.github/workflows/ci-tests.yml

```
@@ -0,0 +1,135 @@
+        uses: ./.github/actions/setup-caches
+        with:
+          cache-type: ${{ matrix.test-type }}
+          os: ${{ runner.os }}
+
+      # application-server tests
+      - name: "application-server ${{ matrix.test-type }} execution"
+        if: startsWith(matrix.test-type, 'application-server-')
+        run: |
+          echo "🧪 Running application-server ${{ matrix.test-type }} tests"
+          cd server/application-server
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)